cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/util/AbstractAppFabricHttpHandler.java

```diff
@@ -298,6 +302,21 @@ protected void getLiveInfo(HttpRequest request, HttpResponder responder, String
     }
   }
 
+  // Adds a schedule to the scheduler as well as to the appspec
+  protected void addSchedule(Scheduler scheduler, Store store, Id.Program programId,
```

Comment:
I think we should remove/move this method. This functionality is needed from a class (AdapterService.java) in a new branch. That class has access to scheduler and store, but not to AbstractAppFabricHttpHandler.